models/repo.go

```diff
@@ -1467,7 +1474,15 @@ func GetUserRepositories(userID int64, private bool, page, pageSize int) ([]*Rep
 // GetUserRepositories returns a list of mirror repositories of given user.
 func GetUserMirrorRepositories(userID int64) ([]*Repository, error) {
 	repos := make([]*Repository, 0, 10)
-	return repos, x.Where("owner_id = ?", userID).And("is_mirror = ?", true).Find(&repos)
+	sess := x.Where("owner_id = ?", userID).And("is_mirror = ?", true).Desc("uid")
```

Comment:
For a better readability and to get the real line / instruction in case of error, please set  XORM instruction per line

eg :

```
sess := x.Where("owner_id = ?", userID).
        And("is_mirror = ?", true).
        Desc("uid")
```
or

```
sess := x.
        Where("owner_id = ?", userID).
        And("is_mirror = ?", true).
        Desc("uid")
```